### PR TITLE
Created Explode Strategy for hydrator

### PR DESCRIPTION
**Pull request by ojhaujjwal**
Too simple and self-explanatory.
Please see the tests.

### CODE REVIEW

File: library/Zend/Stdlib/Hydrator/Strategy/Exception/InvalidArgumentException.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright Copyright (c) 2005-2014 Zend Technologies USA Inc. (http://www.zend.com)
```
Comment (samsonasik) [inline library/Zend/Stdlib/Hydrator/Strategy/Exception/InvalidArgumentException.php]: 2015 ;)

File: library/Zend/Stdlib/Hydrator/Strategy/ExplodeStrategy.php [hunk 2]
```diff
@@ -0,0 +1,125 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright Copyright (c) 2005-2014 Zend Technologies USA Inc. (http://www.zend.com)
```
Comment (samsonasik) [inline library/Zend/Stdlib/Hydrator/Strategy/ExplodeStrategy.php]: 2015 ;)

File: tests/ZendTest/Stdlib/Hydrator/Strategy/ExplodeStrategyTest.php [hunk 3]
```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright Copyright (c) 2005-2014 Zend Technologies USA Inc. (http://www.zend.com)
```
Comment (samsonasik) [inline tests/ZendTest/Stdlib/Hydrator/Strategy/ExplodeStrategyTest.php]: 2015 ;)

File: library/Zend/Stdlib/Hydrator/Strategy/ExplodeStrategy.php [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+        return $this;
+    }
+
+    /**
+     * Sets explode limit
+     *
+     * @param  int  $explodeLimit
+     * @return self
+     */
+    public function setExplodeLimit($explodeLimit)
```
Comment (Ocramius) [inline library/Zend/Stdlib/Hydrator/Strategy/ExplodeStrategy.php]: To be removed

Comment (ojhaujjwal) [inline library/Zend/Stdlib/Hydrator/Strategy/ExplodeStrategy.php]: Removed!

**Comment by Ocramius** [comment]
@ojhaujjwal merged, thanks!

`develop`: zendframework/zf2@526c1d9deaf733323cd750e8aa44b12e90213817

**Pull request closed without merging**